scst-scan/app-scanning-observing.hbs.md

```diff
@@ -0,0 +1,33 @@
+# Observability
+
+This topic provides information to help observe Supply Chain Security Tools - Scan 2.0.
```

Comment:
```suggestion
This topic guides you through observing  Supply Chain Security Tools - Scan 2.0 as it it progresses through a scan.  This will help a user understand the each step of the scanning process.
```